ultimate_cron.module

```diff
@@ -714,15 +341,15 @@ function ultimate_cron_get_module_hooks($module) {
 function ultimate_cron_get_plugin_hooks() {
   $items = array();
   // Invoke hook_cronapi() on the plugins.
-  $plugin_types = ctools_plugin_get_plugin_type_info();
-  foreach ($plugin_types['ultimate_cron'] as $plugin_type => $info) {
-    $plugins = ultimate_cron_plugin_load_all($plugin_type);
-    foreach ($plugins as $plugin) {
-      if ($plugin->isValid()) {
-        $items += $plugin->cronapi();
-      }
-    }
-  }
+  // $plugin_types = ctools_plugin_get_plugin_type_info();
```

Comment:
I'm confused what the purpose of this is. Since we will remove cronapi(), let's just drop this for now.